runtime/vm/ValueTypeHelpers.hpp

```diff
@@ -236,6 +236,24 @@ class VM_ValueTypeHelpers {
 		return acmpResult;
 	}
 
+	static VMINLINE bool
+	isClassRefQtype(J9ConstantPool *ramCP, U_16 cpIndex)
+	{
+		J9ROMStringRef *romStringRef = (J9ROMStringRef *)&ramCP->romConstantPool[cpIndex];
+		J9UTF8 *classNameWrapper = J9ROMSTRINGREF_UTF8DATA(romStringRef);
+		U_16 classNameLength = J9UTF8_LENGTH(classNameWrapper);
```

Comment:
Is there a verifier constraint that the length will never be 0?

Reply:
yes `checkNameImpl` via `bcvCheckClassName` verifies that name length is not zero